src/site/content/en/blog/rendering-on-the-web/index.md

```diff
@@ -2,349 +2,169 @@
-Server rendering can also present interesting decisions when building a [PWA].
-Is it better to use full-page [service worker] caching, or just server-render
-individual pieces of content?
-
-## Client-Side Rendering (CSR)
-
-_Client-side rendering (CSR) means rendering pages directly in the browser using
-JavaScript. All logic, data fetching, templating and routing are handled on the
-client rather than the server._
-
-Client-side rendering can be difficult to get and keep fast for mobile. It can
-approach the performance of pure server-rendering if doing minimal work, keeping
-a [tight JavaScript budget] and delivering value in as few [RTTs] as possible.
-Critical scripts and data can be delivered sooner using `<link rel=preload>`,
-which gets the parser working for you sooner. Patterns
-like [PRPL] are worth evaluating in order to ensure initial and subsequent
-navigations feel instant.
+Solutions for static rendering come in all shapes and sizes. Tools like [Gatsby](https://www.gatsbyjs.org) are designed to make developers feel like their application is being rendered dynamically rather than generated as a build step. Static site generation tools such as [11ty](https://www.11ty.dev/), [Jekyll](https://jekyllrb.com), and [Metalsmith](https://metalsmith.iooperationalizing-node-js-for-server-side-rendering-c5ba718acfc9speedier-server-side-rendering-in-react-16-with-component-caching-e8aa677929b1) embrace their static nature, providing a more template-driven approach.
+
+{% Aside %}
+Abstractions of popular frameworks such as Next.js or Nuxt offer both static rendering and server rendering. This allows developers to opt into static rendering for pages that qualify, or use server rendering for pages that need to be dynamically generated in response to a request.
+{% endAside %}
+
+One of the downsides to static rendering is that individual HTML files must be generated for every possible URL. This can be challenging or even infeasible when you can't predict what those URLs will be ahead of time, or for sites with a large number of unique pages.
+
+React users may be familiar with Gatsby, [Next.js static export](https://nextjs.org/learn/excel/static-html-export/) or [Navi](https://frontarm.com/navi/)—all of these make it convenient to author pages using components. However, it's important to understand the difference between static rendering and prerendering: static rendered pages are interactive without the need to execute much client-side JavaScript, whereas prerendering improves the FCP of a Single Page Application that must be booted on the client in order for pages to be truly interactive.
+
+If you're unsure whether a given solution is static rendering or prerendering, try disabling JavaScript and load the page you want to test. For statically rendered pages, most of the functionality will still exist without JavaScript enabled. For prerendered pages, there may still be some basic functionality like links, but most of the page will be inert.
```

Comment:
Not sure I like this sentence (I realise it's not yours).

For statically rendered and prerendered pages the content will likely still be there. For non-SSR pages there often is little to no content.

In neither case is the page liable to have much "functionality" (if you take that to mean interactivity) without JS.